pkg/router/template/plugin.go

```diff
@@ -99,6 +99,30 @@ func env(name, defaultValue string) string {
 	return defaultValue
 }
 
+// This variable holds the bind string for haproxy as documented
+// http://cbonte.github.io/haproxy-dconv/1.5/configuration.html#4-bind
+var bindLine string = ""
+
+// Due to the fact that haproxy expects a ip:port tuple  I need
+// to build this tuple per IP address
+func createBindline(ipAdress string, port string) string {
+	if bindLine == ""{
+		bindLine = fmt.Sprintf("%s:%s",ipAdress,port)
+	}else {
+		bindLine = fmt.Sprintf("%s,%s:%s",bindLine,ipAdress,port)
+	}
+	return bindLine
+}
+
+func getBindline() string {
+	return bindLine
+}
+
+func resetBindline() string{
+	bindLine = ""
+	return bindLine
```

Comment:
yes.

I get the following error when I remove the return value 

```
panic: can't install method/function "resetBindline" with 0 results
```
https://play.golang.org/p/DbvNfnMzGm

Due to the fact that I'm new to go and there template engine I haven't seen any other solution.